completion/git-machete.completion.bash

```diff
@@ -32,7 +37,7 @@ _git_machete() {
 
   case $cur in
     --branch=*|--onto=*) __gitcomp_nl "$(git machete list managed 2>/dev/null)" "" "${cur##--*=}" ;;
-    --checked-out-since=*) __gitcomp "" ;;
+    --checked-out-since=*|--by=*) __gitcomp_empty "" "${cur##--*=}";;
```

Comment:
Nit:
```suggestion
    --by=*|--checked-out-since=*) __gitcomp_empty "" "${cur##--*=}";;
```
plus move up for abc order

Reply:
Also... why are there params passed to `__gitcomp_empty`? 🤔 they're not picked up in the function anyway